internal/cmdexport/cmdexport_test.go

```diff
@@ -100,6 +105,27 @@ jobs:
             uses: docker://gongpu/kpt:latest
             with:
                 args: fn run . --fn-path functions/
+`,
+	},
+	{
+		description: "exports a Cloud Build pipeline",
+		params: []string{
+			"cloud-build",
+			".",
+			"--fn-path",
```

Comment:
Do you check if `--fn-path` or other paths in the arguments exist somewhere? If not, let's add it.

Reply:
As discussed offline, it would be appropriate to check `--fn-path` will not go beyond the current directory.